### PR TITLE
Feature/228 add and show version number

### LINKED ISSUE #228: show version number somewhere on the web application
As a tech-savvy user, I'd like to know what version of Osmaxx is running on an installation I'm looking at over the web, so that I can
* include this information in bugs I report
* look up what known issues this version has (according to bugs reported by others)
* know what features I can expect (by comparing with milestones and releases on GitHub)

As a developer, I'd like to know what version of Osmaxx is running on an installation I'm looking at, so I can tell whether a scheduled deployment has actually been performed.

In either case, I want to be able to determine this version number without having to access the server backend (e.g. by SSH). Requiring accessing the web backend (`/admin` URL) is acceptable, but would be unfortunate. Optimal would be not even requiring to be logged in.

### PR DESCRIPTION
**Pull request by njordan-hsr**
Closes #228

* added version number
* added separate app to display the version number under /version/

Reviewed by:

* [x] @das-g 

`RUN_E2E=true ./test.sh` all green.

### CODE REVIEW

File: osmaxx-py/config/settings/local.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 MIDDLEWARE_CLASSES += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
 INSTALLED_APPS += ('debug_toolbar', )
 
-INTERNAL_IPS = ('127.0.0.1', )
+INTERNAL_IPS = env.tuple('DJANGO_INTERNAL_IPS', default='127.0.0.1')
```
Comment (das-g) [inline osmaxx-py/config/settings/local.py]: type mismatch

File: osmaxx-py/version/templates/version/show_version.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+{% if DEBUG %}
+    <b style="color: #ff0000">DEBUG IS ON</b>
+{% else %}
+    <b style="color: #00ff00">DEBUG is off</b>
+{% endif %}
+</p>
+<h1>Version in use: {{ version_number }}{% if DEBUG %}-development{% endif %}</h1>
+<p>`-development` is added to the version number if DEBUG is on.</p>
+</body>
+</html>
```
Comment (das-g) [inline osmaxx-py/version/templates/version/show_version.html]: missing newline at end of file

**Comment by das-g** [comment]
If this shall close #228, please update the documentation as mentioned at https://github.com/geometalab/osmaxx/issues/228#issuecomment-146237243

**Comment by njordan-hsr** [comment]
So Update version number under osmaxx-py/osmaxx/__init__.py isn't enough? It's not on this bunch of commits, that might be a bit misleading, since that was an online edit on the develop branch. It's there nonetheless ;-)

**Pull request merged**